### PR TITLE
Python packages.holidays 0.10.1

### PR DESCRIPTION
**Pull request by jluttine**
###### Motivation for this change

Upgrade Python package holidays. Needed to add a few dependencies too.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/convertdate/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    owner = "fitnr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04j8k7a9qndmawy3m345py74y18hw7lb6gc0qp0mr8d68x99xjq0";
+  };
+
+  propagatedBuildInputs = [ pymeeus pytz ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/fitnr/convertdate;
```
Comment (jonringer) [inline pkgs/development/python-modules/convertdate/default.nix]:
```suggestion
    homepage = "https://github.com/fitnr/convertdate";
```

File: pkgs/development/python-modules/pymeeus/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest .
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/architest/pymeeus;
```
Comment (jonringer) [inline pkgs/development/python-modules/pymeeus/default.nix]:
```suggestion
    homepage = "https://github.com/architest/pymeeus";
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

```
[13 built, 3 copied (2.5 MiB), 0.4 MiB DL]
https://github.com/NixOS/nixpkgs/pull/82162
9 package built:
python27Packages.convertdate python27Packages.holidays python27Packages.pymeeus python37Packages.convertdate python37Packages.holidays python37Packages.pymeeus python38Packages.convertdate python38Packages.holidays python38Packages.pymeeus
```

File: pkgs/development/python-modules/convertdate/default.nix (hunk 1 above)
Comment (jluttine) [reply pkgs/development/python-modules/convertdate/default.nix]: Why the double quotes? There are many examples in the manual without the double quotes: https://nixos.org/nixpkgs/manual/#chap-reviewing-contributions

File: pkgs/development/python-modules/pymeeus/default.nix (hunk 2 above)
Comment (jluttine) [reply pkgs/development/python-modules/pymeeus/default.nix]: Why the double quotes? There are many examples in the manual without the double quotes: https://nixos.org/nixpkgs/manual/#chap-reviewing-contributions

File: pkgs/development/python-modules/convertdate/default.nix (hunk 1 above)
Comment (jluttine) [reply pkgs/development/python-modules/convertdate/default.nix]: Got an answer from IRC. Thanks, fixed.

File: pkgs/development/python-modules/pymeeus/default.nix (hunk 2 above)
Comment (jluttine) [reply pkgs/development/python-modules/pymeeus/default.nix]: Got an answer from IRC. Thanks, fixed.

**Comment by jluttine** [comment]
@jonringer Thanks for the review. I made the requested fixes.

**Review by jonringer — APPROVED** [review]
LGTM

```
https://github.com/NixOS/nixpkgs/pull/82162
9 package built:
python27Packages.convertdate python27Packages.holidays python27Packages.pymeeus python37Packages.convertdate python37Packages.holidays python37Packages.pymeeus python38Packages.convertdate python38Packages.holidays python38Packages.pymeeus
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.convertdate python27Packages.holidays python27Packages.pymeeus python37Packages.convertdate python37Packages.holidays python37Packages.pymeeus python38Packages.convertdate python38Packages.holidays python38Packages.pymeeus

**Pull request merged**